src/main/java/frc/robot/Robot.java

```diff
@@ -108,13 +110,48 @@ public void robotInit() {
                 () -> -controller.getLeftX() * SwerveSubsystem.MAX_LINEAR_SPEED));
 
     NamedCommands.registerCommand("stop", swerve.stopWithXCmd().asProxy());
+
+    controller.b().toggleOnTrue(autoAimDemo(() -> swerve.getVelocity()));
```

Comment:
generally avoid toggles, either use a `whenTrue` if cancelling out of it should be impossible, or `whileTrue` if cancelling out of it should be allowed

Reply:
set to `whileTrue`